bytebase/bytebase.com
# feat: add Brand section to About Us page

**This PR:**
Adding Brand kit section to About Us page 

**Steps to test**

- Open the [About us page](https://bytebase-git-update-about-us-page-bytebase-pixelpoint.vercel.app/about/)
- Confirm that everything looks and works as expected
